packages/regl-worldview/src/utils/commandUtils.js

```diff
@@ -122,11 +122,23 @@ export function withPose(command: ReglCommand): ReglCommand {
   };
 }
 
-export function getVertexColors(props: any) {
-  if (!props.colors || !props.colors.length) {
-    return constantRGBAArray(props.points.length, props.color);
+export function getVertexColors({
+  colors,
+  color,
+  points,
+}: {
+  colors?: (Color | Vec4)[],
```

Comment:
we should start using [`$ReadOnlyArray`](https://flow.org/en/docs/types/arrays/#toc-readonlyarray) for these because it will make typechecking easier for users.